float64 		radius
VectorMessage 	seed_frp
float64 		seed_fro
int32 			seed_id
int32 			reference_nbr_id
int32 			formation_id
int32 			formation_count
NeighborMap[] 	neighbor_map
float64 		sensor_error
float64 		communication_error